src/dwarf.cpp

```diff
@@ -704,7 +704,7 @@ pool_string dwarf::implementation::die_identifier(const die& d,
 /**************************************************************************************************/
 
 attribute_value dwarf::implementation::evaluate_exprloc(std::uint32_t expression_size) {
-    std::vector<std::int32_t> stack;
+    std::vector<std::int64_t> stack;
```

Comment:
I _think_ DWARF expression stacks are 32-bits. Is there a reason this was bumped to int64?

Reply:
It was just easier than adding a bunch of casts - but obviously I don't have strong feelings either way.

Reply:
I see. I'm not sure we have any possible ODRVs that are specified with this kind of DWARF expression, so I'm happy to leave things as they are.